mini-projects/movie-rec-system/movie_rec_system/recommendation/recommender.py

```diff
@@ -0,0 +1,77 @@
+import json
+import pandas as pd
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.metrics.pairwise import cosine_similarity
+from recommender_helper import (
+    content_movie_recommender,
+    get_popularity_rmse,
+    get_vote_avg_rmse,
+    get_vote_count_rmse,
+)
+
+
+def connect_duckdb():
```

Comment:
Does this function depend on where/how we store the data?

For instance, if it's in a S3 bucket that we'll access through Lambda, wouldn't we have to adjust this function to account for that?

Because the bucket isn't yet set up, should I leave this as is for now?

Reply:
Can you build this function `connect_duckdb` assuming you run the fastapi instance locally? 

Let's keep things simple

1. First get a local fastapi instance working
2. we upgrade or add new docker container for it
3. then we can modify the functionality to account for aws architecture
4. Add docstrings to your functions
5. Begin writing simple tests to ensure your functions work appropriately

Reply:
This function has been renamed to `get_data` and works appropriately. 

https://github.com/ploomber/sql/pull/100/files#diff-9b1926a7b439d1798b4654dd83de94bab22f71c9bea3f3db147cde8e9f062b85R14-R26